Widget.php

```diff
@@ -61,7 +61,7 @@ public function init()
             throw new InvalidConfigException('The "plugins" property must be an array.');
         }
         if (!isset($this->settings['lang']) && Yii::$app->language !== 'en-US') {
-            $this->settings['lang'] = substr(Yii::$app->language, 0, 2);
+            $this->settings['lang'] = Yii::$app->language;
```

Comment:
This code is required  because `imperavi` language files are named like `en.js`, `fr.js`, 'de.js', without regional suffix. Excluding  a few of them like `pt_br` that must be setted manually in widget setting, beacuse they use `_` instead of `-`.